packages/toolpad-app/public/schemas/AutocompleteOptions.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "array",
+  "items": {
+    "type": ["object", "string"],
+    "additionalProperties": true,
+    "properties": {
+      "label": {
```

Comment:
Wait, shouldn't options be able to have an `id` or value` different than the label?